test/mailers/subscription_mailer_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'katello_test_helper'
+
+module Katello
+  class SubscriptionMailertest < ActiveSupport::TestCase
+    def setup
+      @user = User.current = User.find(users('admin').id)
+
+      FactoryBot.create(:mail_notification,
+                        :name => 'subscription_expiry',
+                        :description => 'A list of Subscriptions expiring in 30 days',
+                        :mailer => 'Katello::SubscriptionMailer',
+                        :method => 'subscription_expiry',
+                        :subscription_type => 'report')
+
+      @user.mail_notifications << MailNotification[:subscription_expiry]
+      @pool = katello_pools(:pool_one)
```

Comment:
@jeremylenz recently created an easy way to get a subscription that's expiring soon with a FactoryBot trait like this ` FactoryBot.build(:katello_pool, :expiring_in_12_days)` This way you don't have to mess with the fixtures.  As written, this test will fail soon because april 15 2020 (the date on the pool fixture) will be behind us!